tests/integration/test_interaction.py

```diff
@@ -581,3 +581,138 @@ def test_keyboard_help(self):
         # Test mouse and keyboard interaction
         self.interact_with_keyboard_help(scroll_down=900)
         self.interact_with_keyboard_help(scroll_down=0, use_keyboard=True)
+
+
+class ZoneAlignInteractionTest(InteractionTestBase, BaseIntegrationTest):
+    """
+    Verifying Drag and Drop XBlock interactions using zone alignment.
+    """
+    PAGE_TITLE = 'Drag and Drop v2'
+    PAGE_ID = 'drag_and_drop_v2'
+
+    def setUp(self):
+        super(ZoneAlignInteractionTest, self).setUp()
+
+    def _get_scenario_xml(self):
+        return self._get_custom_scenario_xml("data/test_zone_align.json")
+
+    def _assert_zone_align_item(self, item_id, zone_id, align, action_key=None):
+        """
+        Test items placed in a zone with the given align setting.
+        Ensure that they are children of the zone.
+        """
+        # parent container has the expected alignment
+        zone_selector = "div[data-zone='%s'] .item-align" % zone_id
+        self.assertEquals(self._get_style(zone_selector, 'textAlign'), align)
+
+        # Items placed in zones with align setting are children of the zone
+        zone_item = '%s .option' % zone_selector
```

Comment:
@pomegranited `zone_item` ⟶ `zone_item_selector`? (since you're using the `_selector` suffix above)